src/ResultsParser.ts

```diff
@@ -211,8 +211,22 @@ export default class ResultsParser {
 
   updateResults(data: { testSuite: any }) {
     if (data.testSuite.tests.length > 0) {
-      this.result.push(data);
-    }
+      const resIndex = this.result.findIndex(res=>(res.testSuite.title==data.testSuite.title))
```

Comment:
could run `yarn lint-fix` and re-commit this file.  Its causing the build to fail and preventing the unit tests from running